### ISSUE TITLE
Cannot invoke "java.util.Map.containsKey(Object)"

### ISSUE THREAD

**Issue by sphings79** [stumper66/SpawnerControl#2]
```
[10:36:29 WARN]: [SpawnerControl] Plugin SpawnerControl v0.95.0 generated an exception while executing task 1871
java.lang.NullPointerException: Cannot invoke "java.util.Map.containsKey(Object)" because "main.wgRegionOptions" is null
	at me.stumper66.spawnercontrol.WorldGuardManager.updateWorlguardOptionsForSpawner(WorldGuardManager.java:62) ~[SpawnerControl-0.95.0.jar:?]
	at me.stumper66.spawnercontrol.processing.UpdateProcessor.evaluateTrackingCriteriaForSpawner(UpdateProcessor.java:142) ~[SpawnerControl-0.95.0.jar:?]
	at me.stumper66.spawnercontrol.processing.UpdateProcessor.processSpawnerOrChunkAdd(UpdateProcessor.java:112) ~[SpawnerControl-0.95.0.jar:?]
	at me.stumper66.spawnercontrol.processing.UpdateProcessor.processUpdates(UpdateProcessor.java:57) ~[SpawnerControl-0.95.0.jar:?]
	at me.stumper66.spawnercontrol.processing.SpawnerProcessor.processSpawners(SpawnerProcessor.java:75) ~[SpawnerControl-0.95.0.jar:?]
	at me.stumper66.spawnercontrol.SpawnerControl$1.run(SpawnerControl.java:77) ~[SpawnerControl-0.95.0.jar:?]
	at org.bukkit.craftbukkit.v1_18_R1.scheduler.CraftTask.run(CraftTask.java:101) ~[paper-1.18.1.jar:git-Paper-144]
	at org.bukkit.craftbukkit.v1_18_R1.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:57) ~[paper-1.18.1.jar:git-Paper-144]
	at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22) ~[paper-1.18.1.jar:git-Paper-144]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
```

```
[10:38:14 INFO]: Checking version, please wait...
[10:38:14 INFO]: This server is running Paper version git-Paper-144 (MC: 1.18.1) (Implementing API version 1.18.1-R0.1-SNAPSHOT) (Git: 87d8ef9)
You are running the latest version
Previous version: git-Paper-141 (MC: 1.18.1)
```

```
[10:38:57 INFO]: SpawnerControl version 0.95.0
[10:38:57 INFO]: Makes spawners under any conditions
[10:38:57 INFO]: Website: https://github.com/stumper66/SlimeSpawners
[10:38:57 INFO]: Author: stumper66
```

**Comment by stumper66** [comment]
Thanks for reporting the issue.  Will look into it.

**Comment by stumper66** [comment]
Hi @sphings79 
This has been resolved in v0.96.0.  It's been posted on the spigotmc page https://www.spigotmc.org/resources/spawnercontrol.98872/
